src/probnum/quad/bq_methods/bayesian_quadrature.py

```diff
@@ -0,0 +1,125 @@
+"""Probabilistic numerical methods for solving integrals."""
+
+from typing import Callable, Dict, Optional, Tuple
+
+import numpy as np
+from scipy.linalg import cho_factor, cho_solve
+
+from probnum.kernels import ExpQuad, Kernel
+from probnum.randvars import Normal
+
+from .._integration_measures import IntegrationMeasure
+from ..kernel_embeddings import KernelEmbedding
+from ..policies import sample_from_measure
+
+
+class BayesianQuadrature:
+    r"""A base class for Bayesian quadrature
```

Comment:
Petty docstring comments:
- A base class for Bayesian quadrature.
- .. math:: F = \int_\Omega f(x) d \mu(x).
- The kernel ... full stop
- The policy ... full stop